roles/gerrit/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Install Java and python-bugzilla
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - java-1.8.0-openjdk
+  - python-bugzilla
+
+- name: Create a user for review
+  user:
+    name: "{{gerrit_user}}"
+    home: "{{gerrit_user_home}}"
+
+
+- name: Check if Gerrit is installed
+  stat:
+    path: "{{gerrit_user}}/{{gerrit_user_home}}"
+    register: gerrit_status
```

Comment:
Wrong indentation, register is at the same level as stat (that's a per task attribute, not a attribute of stat)